docs/changes/12203.bugfix.rst

```diff
@@ -0,0 +1 @@
+Avoid unnecessary transforms through CIRS for AltAz and HADec.
```

Comment:
Presumably you should move this changelog entry to be under `docs/changes/coordinates/`

Reply:
done